pyo3-build-config/src/impl_.rs

```diff
@@ -1530,16 +1535,26 @@ fn default_cross_compile(cross_compile_config: &CrossCompileConfig) -> Result<In
     let implementation = cross_compile_config
         .implementation
         .unwrap_or(PythonImplementation::CPython);
+    let gil_disabled = false;
```

Comment:
so we are still missing `python3.13t` support for `PYO3_CROSS_PYTHON_VERSION`, but I'm running out of energy to implement it today.